Sources/ClientRuntime/Interceptor/DefaultInterceptorContext.swift

```diff
@@ -35,9 +34,13 @@ public class DefaultInterceptorContext<
         self.input
     }
 
-    public func getAttributes() -> AttributesType {
+    public func getAttributes() -> Context {
         return self.attributes
     }
+    
```

Comment:
please take care of the lint issues

Reply:
Maybe we should make these lint issues errors instead of warnings so they fail local `swiftlint` runs and CI